### ISSUE TITLE
Remove connection injection

### ISSUE THREAD

**Issue by nglx** [CambridgeEducation/qualtrics_api#10]
Since the api allows to chain commands like:

```
client.panels.find("panelIdHere").members.create(panel_member)
```

we have to inject the connection to the chained objects.

Let's change the api so the client is not instantiated but configured:

``` ruby
QualtricsAPI.configure do |c|
  c.api_token = 'xxx'
end

QualtricsAPI.panels
```

Leaving the current implementation with client new would be inconsistent in that case (suggesting that we can have multiple clients):

``` ruby
client = QualtricsAPI.new(:api_token => 'xxx')
```

This will simplify the implementation. Makes sense?

**Comment by pallymore** [comment]
yes...please do :)

**Comment by nglx** [comment]
it's implemented in #11
